### PR TITLE
Migrated secret endpoints to raw endpoints in ui

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Description 📣

This PR is part of the new architecture infisical secret manager will be having. This PR makes both e2ee and raw endpoints have same set of argument support. 

Switched all e2ee endpoints to raw endpoints. 

## Type ✨

- [x] Refactor
- [ ] New feature
- [ ] Breaking change
- [ ] Documentation

# Tests 🛠️

```sh
# Here's some code block to paste some code snippets
```

---

- [x] I have read the [contributing guide](https://infisical.com/docs/contributing/getting-started/overview), agreed and acknowledged the [code of conduct](https://infisical.com/docs/contributing/getting-started/code-of-conduct). 📝

### CODE REVIEW

**Comment by akhilmhdh** [comment]
@maidul98 The only issue i see is for users who have yet to migrate will have a breaking dashboard because if bot is also not active then there  secrets won't be pulled

File: frontend/src/hooks/api/secretApprovalRequest/queries.tsx [hunk 1]
```diff
@@ -45,6 +45,78 @@ export const secretApprovalRequestKeys = {
   ]
 };
 
+export const decryptSecrets = (
+  encryptedSecrets: EncryptedSecret[],
+  decryptFileKey: UserWsKeyPair
+) => {
+  const PRIVATE_KEY = localStorage.getItem("PRIVATE_KEY") as string;
```
Comment (maidul98) [inline frontend/src/hooks/api/secretApprovalRequest/queries.tsx:52]: Why is this still from local storage?

**Review by maidul98 — COMMENTED** [review]
I noticed we didn't make point in time recovery raw yet so i assume you'll do that next. I found one bug when deleting multiple secrets from the overview page:

```
infisical-dev-api              | [00:05:09.872] ERROR (152): Cannot read properties of undefined (reading 'secretKey') {"reqId":"req-ek","severity":"ERROR"}
infisical-dev-api              |     err: {
infisical-dev-api              |       "type": "TypeError",
infisical-dev-api              |       "message": "Cannot read properties of undefined (reading 'secretKey')",
infisical-dev-api              |       "stack":
infisical-dev-api              |           TypeError: Cannot read properties of undefined (reading 'secretKey')
infisical-dev-api              |               at /app/src/server/routes/v3/secret-router.ts:6:4133
infisical-dev-api              |               at Array.map (<anonymous>)
infisical-dev-api              |               at Object.handler (/app/src/server/routes/v3/secret-router.ts:6:4070)
infisical-dev-api              |               at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
infisical-dev-api              |     }
infisical-dev-api              | [00:05:09.874] ERROR (152): Cannot read properties of undefined (reading 'secretKey') {"reqId":"req-ek","severity":"ERROR","req":{"method":"DELETE","url":"/api/v3/secrets/batch/raw","hostname":"localhost:8080","remoteAddress":"192.168.176.1","remotePort":33596},"res":{"statusCode":500}}
infisical-dev-api              |     err: {
infisical-dev-api              |       "type": "TypeError",
infisical-dev-api              |       "message": "Cannot read properties of undefined (reading 'secretKey')",
infisical-dev-api              |       "stack":
infisical-dev-api              |           TypeError: Cannot read properties of undefined (reading 'secretKey')
infisical-dev-api              |               at /app/src/server/routes/v3/secret-router.ts:6:4133
infisical-dev-api              |               at Array.map (<anonymous>)
infisical-dev-api              |               at Object.handler (/app/src/server/routes/v3/secret-router.ts:6:4070)
infisical-dev-api              |               at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Triggered from deleting multiple secrets at overview 
```

Otherwise, the changes look good

File: frontend/src/hooks/api/secretApprovalRequest/queries.tsx (hunk 1 above)
Comment (akhilmhdh) [reply frontend/src/hooks/api/secretApprovalRequest/queries.tsx:52]: Because it still used in places. Progressively removing it @maidul98

Comment (maidul98) [reply frontend/src/hooks/api/secretApprovalRequest/queries.tsx:52]: oh okay cool, but before we we'll have it removed right?

**Review by maidul98 — APPROVED** [review]

File: frontend/src/views/SecretMainPage/SecretMainPage.tsx [hunk 2]
```diff
@@ -255,7 +253,6 @@ export const SecretMainPage = () => {
           <>
             <ActionBar
               secrets={secrets}
-              importedSecrets={importedSecrets}
```
Comment (sheensantoscapadngan) [inline frontend/src/views/SecretMainPage/SecretMainPage.tsx:258]: intentional?

**Review by sheensantoscapadngan — APPROVED** [review]

**Pull request merged**